### DISCUSSION TITLE
本地搜索启用了，但是没效果

### DISCUSSION THREAD

**Discussion by xWenChen** [hugo-next/hugo-theme-next#128] [Q&A]
遇到这种问题，怎么处理呢？

![image](https://user-images.githubusercontent.com/36958243/276898179-a3afb75a-c0c2-446a-8d30-d919026d29de.png)

![](https://user-images.githubusercontent.com/36958243/276898314-fff561de-a07c-4ba7-bc68-301d0342c817.png)

![](https://user-images.githubusercontent.com/36958243/276898458-8674987e-49ef-4889-b977-54b599ae24e8.png)

**Comment by xWenChen** [comment]
调了下控制台，发现是这个错误。

![image](https://github.com/hugo-next/hugo-theme-next/assets/36958243/4559429a-7e2a-46bc-a6e3-54a99abe3dea)

请教了下公司的前端大佬，这么改就 OK 了。获取内容的时候加了个问号。

![image](https://github.com/hugo-next/hugo-theme-next/assets/36958243/1a1a18af-4ba9-4178-a815-b6b3d619a165)

**Reply by elkan1788** [reply]
能否提供下你的使用环境呢？比如浏览器版本，目前在本机的微软Edge浏览器上测试暂没有发现此问题。

**Reply by xWenChen** [reply]
![image](https://github.com/hugo-next/hugo-theme-next/assets/36958243/089be981-c984-4f4a-ac04-2c318dc55d64)

**Comment by litanid** [comment]
我的本地搜索也是一样不起作用，加了问号也是不管作用。都不知怎么搞了。

**Reply by elkan1788** [reply]
@litanid 你这个问题比较麻烦一些，似乎是文章内容有关键字，导致生成的本地索引文件有异常无法解析
<img width="1818" height="331" alt="image" src="https://github.com/user-attachments/assets/13f9e66d-4098-4cae-8e1d-eb7936489393" />

建议还是使用在线的Algolia搜索，那个API解析问题后面抽空修复一下

**Comment by litanid** [comment]
终于搞定了，原来是文章标题里之前有一个特殊字符`&`，我把它改成了 and 就好了。

**Comment by litanid** [comment]
多谢指出修改方向。
